gateway/server.go

```diff
@@ -1245,6 +1245,18 @@ func (gw *Gateway) initialiseSystem() error {
 	overrideTykErrors(gw)
 
 	gwConfig := gw.GetConfig()
+	// Initialize the appropriate log formatter
+	if gwConfig.LogFormat != "" {
+		format := strings.ToLower(gwConfig.LogFormat)
+		log.Formatter = logger.NewFormatter(gwConfig.LogFormat)
+		switch format {
```

Comment:
Most of this is redundant:

- dont use strings.ToLower, it should accept 'json', not 'JSON', etc.
- all of this needs to be something like 1 lines:

```
log.Formatter = logger.NewFormatter(gwConfig.LogFormat)
```

If you want to print to the log, do it once and pass %s with a sanitized LogFormat; ("standard", "json"), using log.

You're using `log` here (global), and `mainLog`? Are these global loggers all covered by setting the formatter just on `log`?